docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -12,6 +12,71 @@ ms.devlang: azurecli
 ---
 
 # Azure CLI release notes
+## February 26, 2019
+
+Version 2.0.59
+
+### Core
+
+* Fix issue where in some instances using `--subscription NAME` would throw an exception.
+
+### ACR
+
+* Add '--target' parameter for 'az acr build', 'az acr task create' and 'az acr task update' commands.
+* Improve error handling for runtime commands when not logged into Azure.
+
+### ACS
+
+* add listen-address option to "az aks port-forward"
+
+### AppService
+
+* functionapp: az functionapp devops-build, new command created
+
+### Batch
```

Comment:
Please edit this section so that it follows the release notes standards:

* User-facing changes only
* No links to external release notes

Reply:
Done.